### PR TITLE
Script improvements

### PR DESCRIPTION
**Pull request by Kidswiss**
Ensure that all host packages are updated by copying the repo list to the host.

Also inlcude upstream docker repos so it will be updated, too.

### CODE REVIEW

File: update-packages.sh [hunk 1]
```diff
@@ -10,6 +10,16 @@ log() {
   echo "$(date -u +"%Y-%m-%d %H:%M:%S UTC") [${1}] - ${2}"
 }
 
+apt_get_update() {
```
Comment (simu) [inline update-packages.sh]: Is there a reason why this function is in the update script again?

File: testing/testpod.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+kind: Job
+metadata:
+  name: hello
+spec:
+  template:
+    # This is the pod template
+    spec:
+      containers:
+      - name: hello
+        image: kidswiss/ubuntu-update:05052020
```
Comment (simu) [inline testing/testpod.yaml:11]: would probably make sense to point this to docker.io/vshn now?

**Review by simu — CHANGES_REQUESTED** [review]
LGTM overall

**Review by simu — CHANGES_REQUESTED** [review]
LGTM overall, I've left a couple small comments

File: update-packages.sh (hunk 1 above)
Comment (Kidswiss) [reply update-packages.sh]: probably forgot to remove it again after debugging the packages that didn't get updated correctly.

**Review by simu — APPROVED** [review]

**Pull request merged**